async-request-reply/src/AsyncOperationStatusChecker.cs

```diff
@@ -1,29 +1,30 @@
-using System;
-using System.IO;
-using System.Linq;
-using System.Threading.Tasks;
+using Azure.Storage.Blobs;
 using Azure.Storage.Blobs.Specialized;
+using Azure.Storage.Sas;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.Azure.WebJobs;
-using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Azure.Functions.Worker;
 using Microsoft.Extensions.Logging;
 
-namespace Contoso
+namespace asyncpattern
```

Comment:
Just to follow the naming conventions of C#, can you make namespace `Asyncpattern` everywhere instead?

Reply:
Working on this. Running a new end-to-end test

Reply:
done